stable/concourse/more-config.yaml

```diff
@@ -1,8 +0,0 @@
-web:
```

Comment:
damn, this one was me being stupid and using `git add --all .`. Lesson learned 👍